dataland-frontend/src/components/wrapper/AuthorizationWrapper.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div v-if="hasUserRequiredRole">
+  <div v-if="hasUserRequiredRole || isUserDataOwner">
     <slot></slot>
   </div>
+  <div v-if="!hasUserRequiredRole && waitingForData" class="d-center-div text-center px-7 py-4">
+    <p class="font-medium text-xl">Checking for data ownership...</p>
+    <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+  </div>
   <TheContent v-else class="paper-section flex">
```

Comment:
I cannot visit this page, even though I am the data owner of this companyId:

![image](https://github.com/d-fine/Dataland/assets/99255296/2b399eb2-040c-41db-8033-bf98ae454923)

Reply:
ok this and the edit page finding is strange as it did work sometime ago
i will check why this is broken now

Reply:
Still looking into this

Reply:
Should be working again

Reply:
Jetzt kann ich die Seite zwar besuchen, wenn ich aber auf ein "Create Dataset" button klicke, sehe ich für ein paar Sekunden, dass ich keine permission habe.
anschließend erscheint die page jedoch

Reply:
Ja das liegt daran, dass er im Hintergrnud deine Ownership prüft. Es sollte dabei aber ein Loading Spinner zu sehen sein.

Reply:
Hier ist das hingegen scheinbar gefixt worden durch deine Code-Änderungen.
Ich sehe nur noch den "check ownership" spinner, so wie es sein sollte 👍